test/suites/clustering.sh

```diff
@@ -2514,6 +2514,15 @@ test_clustering_fan() {
   LXD_DIR="${LXD_ONE_DIR}" lxc exec c1 -- ping -c2 -W5 "${IP_C2}"
   LXD_DIR="${LXD_ONE_DIR}" lxc exec c2 -- ping -c2 -W5 "${IP_C1}"
 
+  echo "Check that the DHCP leases are cleaned up post-migration"
+  lxc stop c1
+  LXD_DIR="${LXD_ONE_DIR}" lxc move c1 --target node2
+  lxc start c1
+  if grep " c1" "${LXD_ONE_DIR}/networks/${fanbridge}/dnsmasq.leases" ; then
```

Comment:
```suggestion
  if grep -qF " c1 " "${LXD_ONE_DIR}/networks/${fanbridge}/dnsmasq.leases" ; then
```

This slight tweak is to avoid any risk of matching with any IPv6 hex (granted FAN doesn't support v6) or a MAC address.